cloudvolume/storage/storage_interfaces.py

```diff
@@ -318,6 +318,8 @@ def get_file(self, file_path, start=None, end=None):
       resp = requests.get(key, headers=headers)
     else:
       resp = requests.get(key)
+      if resp.headers.get('Content-Encoding') == 'br':
```

Comment:
Might be good to leave a comment here with the link to the requests and urllib3 issues.